docs/building-apps.md

```diff
@@ -218,6 +219,25 @@ my-app/
   webos-meta/
 ```
 
+## [contenthash] Injection
+
+For supporting [`Caching`](https://webpack.kr/guides/caching), Enact CLI supports adding the hash of the chunk. You can use the [`contenthash`](https://webpack.js.org/configuration/output/#template-strings) substitutions setting to define the names of your output files.
```

Comment:
I can't see `content-hash` option in this document. Please re-write it.

Reply:
OK